src/server/api/schema.js

```diff
@@ -1155,11 +1165,15 @@ const rootResolvers = {
         return user;
       }
     },
+    currentUserWithAccess: async (_, { organizationId, role }, { user }) => {
```

Comment:
I don't thiink we need this -- we can just query:
`currentUser { roles(organizationId:$orgId) }`